index.html

```diff
@@ -5,7 +5,7 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <link rel="stylesheet" href="style.css" />
-    <title>Porfolio Website</title>
+    <title>Portfolio Website</title>
```

Comment:
- [ ] Please kindly change the title to more professional detailed content. For example, you can use 'Said's Portfolio' or 'Said full-stack developer.' 🙌🏼

Reply:
fixed